cmd/osv-scanner/main_test.go

```diff
@@ -176,7 +176,7 @@ func TestRun(t *testing.T) {
 		{
 			name: "Scan locks-many",
 			args: []string{"", "./fixtures/locks-many"},
-			exit: 0,
+			exit: 1,
```

Comment:
@another-rex we may need to update the fixtures to make sure no vulnerabilities. will you be happy to help on this?